packages/matter.js/src/util/Cache.ts

```diff
@@ -9,6 +9,7 @@
 import { Time, Timer } from "../time/Time.js";
 
 export class Cache<T> {
+    private readonly knownKeys = new Set<string>();
```

Comment:
What does tracking the keys separately from the value map do?

Reply:
The problem was that a "clear" on the cache cleared all values ... but also all keys, so the cache forgot with keys are basically maybe there still because they were already cached. So the "keys()" method was also returning nothing after clearing. So this change is remembering keys so that keys() an also be used after clearing all "values". Makes sense?

Reply:
yeah makes sense